command/repo.go

```diff
@@ -2,19 +2,30 @@ package command
 
 import (
 	"fmt"
+	"net/url"
 	"os"
 	"strings"
+	"time"
 
+	"github.com/cli/cli/api"
 	"github.com/cli/cli/git"
 	"github.com/cli/cli/internal/ghrepo"
 	"github.com/cli/cli/utils"
+
+	"github.com/AlecAivazis/survey/v2"
 	"github.com/spf13/cobra"
 )
 
 func init() {
 	RootCmd.AddCommand(repoCmd)
 	repoCmd.AddCommand(repoCloneCmd)
 	repoCmd.AddCommand(repoViewCmd)
+	repoCmd.AddCommand(repoForkCmd)
+
+	repoForkCmd.Flags().String("clone", "prompt", "Clone fork. Pass false to not clone fork.")
+	repoForkCmd.Flags().String("remote", "prompt", "Add remote for fork. Pass false to not add remote.")
+	repoForkCmd.Flags().Lookup("clone").NoOptDefVal = "true"
+	repoForkCmd.Flags().Lookup("remote").NoOptDefVal = "true"
```

Comment:
TIL `NoOptDefVal`!

I wonder, though, if the users will understand that the default value for `--clone` _omitted_ is "prompt" (as indicated in help docs), but that the default value for `--clone` _passed_ without value is "true" 🤔

Reply:
@mislav Yep, same ambivalence as you here, and @ampinsk and @vilmibm went back and forth on it starting here: https://github.com/cli/cli/pull/549#discussion_r384823866

It didn't seem like anything we thought of was obviously very elegant, but this feels like a reasonable starting point at least to me and we can always update it if we see evidence of confusion.